package.json

```diff
@@ -75,7 +75,7 @@
     "universal-cookie": "4.0.2"
   },
   "devDependencies": {
-    "@edx/frontend-build": "1.2.2",
+    "@edx/frontend-build": "^1.3.1",
```

Comment:
Oh one comment. Do we want to pin this to be in line with everything else?

Reply:
All set.